src/harness/certs/generate_fake_certs.sh

```diff
@@ -151,6 +147,31 @@ openssl ca -cert unknown_ca.cert -keyfile private/unknown_ca.key -in unknown_dev
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
 
+echo "\n\nGenerate 'dp' certificate/key"
```

Comment:
Is this change related to this PR?

Reply:
No, it's related to   DP security changes. Removed.